specification/compute/resource-manager/Microsoft.Compute/stable/2017-12-01/compute.json

```diff
@@ -2689,6 +2689,110 @@
         "x-ms-long-running-operation": true
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/virtualMachineScaleSets/{vmScaleSetName}/redeploy": {
+      "post": {
```

Comment:
Please provide `x-ms-examples` for each of these operations.

Reply:
This is just another standard operation. Similar ones already exist in the single VM case without examples.

Reply:
I understand but we need examples to approve new changes, it helps us validate the examples with the operations described in the spec.